src/m365/planner/Plan.ts

```diff
@@ -0,0 +1,7 @@
+export interface Plan {
```

Comment:
Instead of introducing our own type, let's use the typings from the @microsoft/microsoft-graph-types package.